os-server/src/main/java/app/openschool/usermanagement/exceptions/EmailAlreadyExistException.java

```diff
@@ -0,0 +1,10 @@
+package app.openschool.usermanagement.exceptions;
+
+/** Useful Javadoc. */
+public class EmailAlreadyExistException extends RuntimeException {
```

Comment:
where this exception is handled?

Reply:
@RestControllerAdvice 
app.openschool.usermanagement.exceptions.UserExceptionHandling class.